### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by duynq**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,47 +1,23 @@
...
-
-# Use jquery as the JavaScript library
 gem 'jquery-rails'
-# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
 gem 'turbolinks'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.0'
-# bundle exec rake doc:rails generates the API under doc/api.
+gem 'bootstrap-sass', '~> 3.3.6'
 gem 'sdoc', '~> 0.4.0', group: :doc
```
Comment (nhunght) [inline Gemfile]: ""

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer" >
+  <div class="container" >
+  <p class="text-muted" >FramgiaCRB</p>
```
Comment (nhunght) [inline app/views/layouts/_footer.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<header class="navbar navbar-default" >
+  <div class="contanier">
+    <div class="navbar-header" >
+      <%= link_to "FramgiaCRB", "#", class: "navbar-brand" %>
```
Comment (nhunght) [inline app/views/layouts/_header.html.erb]: i18n

File: config/application.rb [hunk 4]
```diff
@@ -17,8 +17,8 @@ class Application < Rails::Application
     # config.time_zone = 'Central Time (US & Canada)'
 
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
+    config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
+    config.i18n.default_locale = :en
```
Comment (nhunght) [inline config/application.rb]: " "

File: config/database.yml.example [hunk 5]
```diff
@@ -1,25 +0,0 @@
...
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  <<: *default
-  database: db/test.sqlite3
-
-production:
-  <<: *default
-  database: db/production.sqlite3
```
Comment (nhunght) [inline config/database.yml.example]: để nguyên file này nhé

File: app/views/layouts/_footer.html.erb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer" >
```
Comment (tqtrung09) [inline app/views/layouts/_footer.html.erb]: No space before >

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+  <div class="container">
+  <p class="text-muted" ><%= t "framgia_crb" %></p>
```
Comment (daodat18plus) [inline app/views/layouts/_footer.html.erb]: lùi vào 1 tab

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -7,8 +7,13 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
-
+<%= render "layouts/header" %>
```
Comment (daodat18plus) [inline app/views/layouts/application.html.erb]: lùi vào 1 tab

**Comment by nhunght** [comment]
OK

File: app/assets/stylesheets/static_pages.scss [hunk 9]
```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0px;
+  background: #FFF;
+}
+
+#header{
+
```
Comment (tqtrung09) [inline app/assets/stylesheets/static_pages.scss]: remove line

File: app/views/layouts/_footer.html.erb [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+  <div class="container">
+    <p class="text-muted" ><%= t "framgia_crb" %></p>
```
Comment (tqtrung09) [inline app/views/layouts/_footer.html.erb]: No space before >

File: app/views/layouts/application.html.erb [hunk 11]
```diff
@@ -1,14 +1,19 @@
...
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= yield(:title) %></title>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (tqtrung09) [inline app/views/layouts/application.html.erb]: convention

**Comment by tqtrung09** [comment]
Ok

File: app/assets/javascripts/static_pages.coffee [hunk 12]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (tqtrung09) [inline app/assets/javascripts/static_pages.coffee]: Delete file

File: Gemfile.lock [hunk 13]
```diff
@@ -170,8 +170,8 @@ GEM
       json (~> 1.7, >= 1.7.7)
       rdoc (~> 4.0)
     slop (3.6.0)
-    spring (1.6.4)
-    sprockets (3.5.2)
+    spring (1.7.0)
+    sprockets (3.6.0)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.0.4)
```
Comment (nhunght) [inline Gemfile.lock]: checkout this file

File: app/assets/javascripts/static_pages.coffee (hunk 12 above)
Comment (nhunght) [inline app/assets/javascripts/static_pages.coffee]: checkout this file

File: app/assets/stylesheets/static_pages.scss [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+      display: block;
+      text-align: center;
+      padding: 18px 16px;
+      text-decoration: none;
+      border-right: none;
+    }
+  }
+}
+
+.footer{
```
Comment (tqtrung09) [inline app/assets/stylesheets/static_pages.scss]: have space before {

**Comment by nhunght** [comment]
Ok

File: app/views/layouts/application.html.erb [hunk 15]
```diff
@@ -1,14 +1,20 @@
...
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= yield(:title) %></title>
```
Comment (tqtrung09) [inline app/views/layouts/application.html.erb]: remove ()

**Comment by nhunght** [comment]
OK

**Comment by tqtrung09** [comment]
OK

**Comment by daodat18plus** [comment]
OK

**Pull request merged**